### PR TITLE
[testing] clean-up test_unary_ufuncs.py

### PR DESCRIPTION
**Pull request by kshitij12345**
Some clean-ups

### CODE REVIEW

File: test/test_unary_ufuncs.py [hunk 1]
```diff
@@ -882,13 +882,6 @@ def test_unary_out_op_mem_overlap(self, device, dtype):
             self.check_internal_mem_overlap(in_fn, 1, dtype, dev,
                                             expected_failure=not has_internal_mem_overlap_check)
 
-    # TODO: review with ceil opinfo
-    @onlyCUDA
-    def test_ceil_out_mismatch(self, device):
```
Comment (kshitij12345) [inline test/test_unary_ufuncs.py:887]: OpInfo Entry:
https://github.com/pytorch/pytorch/blob/9ec6883442df8ead765941441854a5518e35c69c/torch/testing/_internal/common_methods_invocations.py#L3893-L3895

Relevant OpInfo test:
https://github.com/pytorch/pytorch/blob/9ec6883442df8ead765941441854a5518e35c69c/test/test_ops.py#L549-L556

File: test/test_unary_ufuncs.py [hunk 2]
```diff
@@ -1241,32 +1234,6 @@ def test_abs_signed_zero(self, device, dtype):
         for v in inp:
             self.assertGreater(math.copysign(1.0, v), 0.0)
 
-    # TODO: rationalize with abs testing and verify absolute is tested as an alias
-    @dtypes(torch.float)
-    def test_absolute(self, device, dtype):
```
Comment (kshitij12345) [inline test/test_unary_ufuncs.py:1246]: OpInfo Entry:
https://github.com/pytorch/pytorch/blob/9ec6883442df8ead765941441854a5518e35c69c/torch/testing/_internal/common_methods_invocations.py#L3529-L3531

Relevant OpInfo test:
https://github.com/pytorch/pytorch/blob/9ec6883442df8ead765941441854a5518e35c69c/test/test_ops.py#L441-L444

File: test/test_unary_ufuncs.py [hunk 3]
```diff
@@ -1511,36 +1478,6 @@ def test_exp(self, device, dtype):
                 # Ensure we are notified when NumPy changes its behavior
                 self.compare_with_numpy(torch.exp, np.exp, nan_real_inf_imag_in)
 
-    # This function tests that a nan value is returned for input values not in domain
-    @dtypes(torch.float32, torch.float64)
-    def test_acosh_domain_float(self, device, dtype):
```
Comment (kshitij12345) [inline test/test_unary_ufuncs.py:1516]: OpInfo Entry:
https://github.com/pytorch/pytorch/blob/9ec6883442df8ead765941441854a5518e35c69c/torch/testing/_internal/common_methods_invocations.py#L3578-L3582

Relevant OpInfo test:
https://github.com/pytorch/pytorch/blob/9ec6883442df8ead765941441854a5518e35c69c/test/test_unary_ufuncs.py#L212-L214

File: test/test_unary_ufuncs.py [hunk 4]
```diff
@@ -1511,36 +1478,6 @@ def test_exp(self, device, dtype):
...
-        nan_mask = torch.tensor([True, False, True, True, True, False], device=device)
-        inf_mask = torch.tensor([False, False, False, False, False, True], device=device)
-        self.assertEqual(torch.isnan(torch.acosh(sample)), nan_mask)
-        self.assertEqual(torch.isnan(sample.acosh()), nan_mask)
-        self.assertEqual(torch.isinf(torch.acosh(sample)), inf_mask)
-        self.assertEqual(torch.isinf(sample.acosh()), inf_mask)
-
-    # This function tests that a nan value is returned for input values not in domain
-    @dtypes(torch.float32, torch.float64)
-    def test_atanh_domain_float(self, device, dtype):
```
Comment (kshitij12345) [inline test/test_unary_ufuncs.py:1530]: OpInfo Entry:
https://github.com/pytorch/pytorch/blob/9ec6883442df8ead765941441854a5518e35c69c/torch/testing/_internal/common_methods_invocations.py#L3855-L3859

Relevant OpInfo test:
https://github.com/pytorch/pytorch/blob/9ec6883442df8ead765941441854a5518e35c69c/test/test_unary_ufuncs.py#L212-L214

**Review by mruberry — APPROVED** [review]
Thanks for the cleanup, @kshitij12345!

**Pull request closed without merging**